apps/web/src/pages/Pools/PoolAdvertisementPage/components/LanguageRequirementDialog.tsx

```diff
@@ -0,0 +1,461 @@
+                  {intl.formatMessage({
+                    defaultMessage: `"A" is the lowest level of bilingual ability and indicates a beginner level.`,
+                    id: "qD0lyZ",
+                    description:
+                      "Proficiency level on language requirements dialog",
+                  })}
+                </li>
+                <li>
+                  {intl.formatMessage({
+                    defaultMessage: `"B" indicates an intermediate level.`,
+                    id: "DoZebI",
+                    description:
+                      "Proficiency level on language requirements dialog",
+                  })}
+                </li>
+                <li>
+                  {intl.formatMessage({
+                    defaultMessage: `"C" is the highest and indicates an advanced level.`,
+                    id: "e9jncy",
+                    description:
+                      "Proficiency level on language requirements dialog",
+                  })}
+                </li>
+              </ul>
+            </div>
+            <div
+              data-h2-display="base(grid)"
+              data-h2-gap="base(x.5)"
+              data-h2-margin-bottom="base(x.5)"
+            >
+              <p>
+                {intl.formatMessage({
+                  defaultMessage:
+                    "Second language proficiency is also determined for three related skills",
+                  id: "HJTr0Q",
+                  description:
+                    "Second language proficiency related skills list fo language requirements dialog",
+                }) + intl.formatMessage(commonMessages.dividingColon)}
+              </p>
+              <ul data-h2-display="base(grid)" data-h2-gap="base(x.5)">
+                <li>
+                  {intl.formatMessage({
+                    defaultMessage: "Reading (or Written Comprehension)",
+                    id: "1XsXrt",
+                    description:
+                      "Proficiency level related skill on language requirements dialog",
+                  })}
+                </li>
+                <li>
+                  {intl.formatMessage({
+                    defaultMessage: "Writing (or Written Expression)",
+                    id: "imNqyO",
+                    description:
+                      "Proficiency level related skill on language requirements dialog",
+                  })}
+                </li>
+                <li>
+                  {intl.formatMessage({
+                    defaultMessage: "Speaking (or Oral Proficiency)",
+                    id: "2hc9V9",
+                    description:
+                      "Proficiency level related skill on language requirements dialog",
+                  })}
+                </li>
+              </ul>
+            </div>
+            <div
+              data-h2-display="base(grid)"
+              data-h2-gap="base(x.5)"
+              data-h2-margin-bottom="base(x1.5)"
+            >
```

Comment:
This spacing does not match the other elements.

![screenshot_17-Jun-2024_09-39-1718631553](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/d1e19cc1-ce9e-4fa8-b9ff-ef0f0e4b9781)

Reply:
That's how it looks in the design.

Reply:
The design had that extra space, but it was a mistake. Fixed by [7957bec](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10628/commits/7957bec46d036ea078394010540617660e66b9df)